planning/behavior_path_planner/include/behavior_path_planner/scene_module/lane_change/base_class.hpp

```diff
@@ -118,16 +112,91 @@ class LaneChangeBase
 
   const CollisionCheckDebugMap & getDebugData() const { return object_debug_; }
 
-  bool isAbortState() const
+  ModuleStatus updateState()
```

Comment:
The module status should be on the interface side.

The reason is that the implementation side don't require this `ModuleStatus` knowledge to work properly.